### DISCUSSION TITLE
Whitelabel email reports

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#148] [Anything Else]
"For freelancers, agencies who want to send email reports to clients with their own branding."

([imported from the old roadmap](https://plausible.nolt.io/108))

**Comment by metmarkosaric** [comment]
Another vote for this from an agency sent to us via email.

**Comment by swarnab99** [comment]
What is the progress of this feature?

**Reply by metmarkosaric** [reply]
There are no short term plans that I can tell you about. We are a small team with limited development resources so need to focus on most frequently requested features first. We have some big ongoing projects such as the Google Analytics import that are taking all of our time right now. Thanks for understanding!

**Comment by LauGau** [comment]
Could be nice, yes :)

**Comment by winchweb** [comment]
Just a note, I won't be able to use Plausible 'for real' until I can at least replace the Plausible logo at the top of the email reports I'd be sending to clients - I don't mind Plausible stuff at the end/bottom, but it would cause confusion if they suddenly get emails from some organisation they've never heard of 😀
Be great to switch over to Plausible though... nice simple & straightforward reports.

**Comment by swarnab99** [comment]
Hi, we have been using Plausible for more than one year, now we really need this feature it will help us to share weekly reports with our clients with our brand name and logo instead of plausible. 

name: Favfly
email: hellofavfly@gmail.com
